controllers/core_alerting_controller_test.go

```diff
@@ -213,4 +216,146 @@ var _ = Describe("Alerting Controller", Ordered, Label("controller", "slow"), fu
 			})
 		})
 	})
+
+	XContext("cortex alertmanager", func() {
```

Comment:
Is this Xed due to flakes?

Reply:
Disabled for now because its inactive until adminv2 api is implemented